sk-usbhid.c

```diff
@@ -985,17 +995,25 @@ sk_sign(uint32_t alg, const uint8_t *data, size_t datalen,
 	*sign_response = NULL;
 	if (check_sign_load_resident_options(options, &device) != 0)
 		goto out; /* error already logged */
-	/* hash data to be signed before it goes to the security key */
```

Comment:
Pleave move this to #else block on line 1012

Reply:
Addressed in 6bd15af424.